linalg/hypre_parcsr.cpp

```diff
@@ -1560,7 +1560,8 @@ void hypre_ParCSRMatrixAbsMatvecT(hypre_ParCSRMatrix *A,
 
    if (num_cols_offd)
    {
-#if MFEM_HYPRE_VERSION >= 21100
+#if MFEM_HYPRE_VERSION >= 21100 && 0
```

Comment:
@artv3, can we take out the `&& 0`?

Reply:
@v-dobrev suggested I commented it out. What do you think @v-dobrev ?